new-bot/src/bot/features/download-video.ts

```
@@ -31,13 +31,41 @@
   let service: 'yt' | 'ig' | 'tw' | 'vk' | 'other' = 'other';
   try {
     ctx.logger.debug(`Processing URL: ${url}`);
+    
+    // Detect and resolve shorthand Instagram URLs by checking inclusion
+    if (url.includes("instagram.com/share/reel/")) {
+      // Ensure URL ends with a slash
+      const urlToResolve = url.endsWith('/') ? url : `${url}/`;
+      const resolvedUrl = await resolveInstagramShorthandUrl(urlToResolve);
+      if (resolvedUrl) {
+        ctx.logger.debug(`Resolved shorthand Instagram URL to: ${resolvedUrl}`);
+        url = resolvedUrl;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)